### DISCUSSION TITLE
Need to find a category for a drum pattern library project

### DISCUSSION THREAD

**Discussion by Babali42** [webprofusion/OpenAudio#155] [General]
I am currently working on www.drumbeatrepo.com a drum pattern library. Repo is on my github.

For now it is only editable via PR but in the future I will open it to non-developers by creating an API and a database.
The project will remain open-source.

Is this relevant in this repository? If so, in which category would you store it!

Thanks in advance

**Comment by webprofusion-chrisc** [comment]
Initially it probably fits under Apps or Open Data Resources. Probably best under Apps for now, which would be an edit to this file: https://github.com/webprofusion/OpenAudio/blob/master/data/apps.json
